addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/ahamodel/TemperatureModel.java

```diff
@@ -30,24 +31,30 @@
     private BigDecimal offset;
 
     public BigDecimal getCelsius() {
-        return celsius != null ? celsius.multiply(TEMP_FACTOR) : BigDecimal.ZERO;
```

Comment:
It feels rather bizarre if this methods returns something which is not converted to celsius yet, what is the value?